Sprint-1/exercises/paths.js

```diff
@@ -1,18 +1,22 @@
-// The diagram below shows the different names for parts of a file path on a Unix operating system
-
-// ┌─────────────────────┬────────────┐
-// │          dir        │    base    │
-// ├──────┬              ├──────┬─────┤
-// │ root │              │ name │ ext │
-// "  /    home/user/dir / file  .txt "
-// └──────┴──────────────┴──────┴─────┘
-
-// (All spaces in the "" line should be ignored. They are purely for formatting.)
-
-const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
-const lastSlashIndex = filePath.lastIndexOf("/");
-const base = filePath.slice(lastSlashIndex + 1);
-console.log(`The base part of ${filePath} is ${base}`);
-
-// Create a variable to store the dir part of the filePath variable
-// Create a variable to store the ext part of the variable
+// The diagram below shows the different names for parts of a file path on a Unix operating system
+
+// ┌─────────────────────┬────────────┐
+// │          dir        │    base    │
+// ├──────┬              ├──────┬─────┤
+// │ root │              │ name │ ext │
+// "  /    home/user/dir / file  .txt "
+// └──────┴──────────────┴──────┴─────┘
+
+// (All spaces in the "" line should be ignored. They are purely for formatting.)
+
+const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
+const lastSlashIndex = filePath.lastIndexOf("/");
+const base = filePath.slice(lastSlashIndex + 1);
+console.log(`The base part of ${filePath} is ${base}`);
+
+// Create a variable to store the dir part of the filePath variable

```

Comment:
When writing code, it's important to think about other software developers who might be reading our code in the future. One way to make our code more readable is to use camel-case variable names. You can read more about what this means here: https://developer.mozilla.org/en-US/docs/Glossary/Camel_case
Can you try changing your variable names to camel case?